modules/drivers/j2k/openjpeg/wscript

```diff
@@ -16,6 +16,11 @@ def options(opt):
 def configure(conf):
     j2kLayer = Options.options.j2k_layer
     j2kHome = Options.options.j2k_home
+
+    if j2kLayer == 'openjpeg':
```

Comment:
So I think this whole flow would be a little more clear (some of this pre-dates your changes) if it was...

```
if j2kLayer == 'openjpeg':
    HAVE_J2K = True
    MAKE_J2K = not j2kHome

    if MAKE_J2K:
        MAKE_OPENJPEG = True
        <existing stuff>
    else:
        <existing stuff>
```

Reply:
I've simplified the logic. MAKE_J2K should not get set to False, because the parent wscript also uses it to decide whether to build Jasper. I just set it here because it makes all the environment checking look a little more explicit, and it won't ever be the case where `j2kLayer == 'openjpeg'`, `j2kHome` is set, and we still want to build Jasper.